src/useFetchyeContext.js

```diff
@@ -0,0 +1,38 @@
+import { useContext, useReducer } from 'react';
```

Comment:
We will need license headers in this file.

Reply:
Good catch.

Reply:
Good catch.

Reply:
Fixed https://github.com/americanexpress/fetchye/pull/6/commits/652c29cc3745860fd9ce46810bc0d56a008a4898